actions/trigger.js

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+
+const triggerDuneQuery = async (context, event) => {
+  const bid_event_hash = await context.storage.getJson('BID_EVENT_HASH')
```

Comment:
now, since exactly this one is deterministic, I'd prefer to have it in the source :see_no_evil: example how that looks at Safe in a contract context: https://github.com/safe-global/safe-contracts/blob/main/contracts/libraries/SignMessageLib.sol#L12. This makes perfectly clear inside the source code what that hash is. And a BidEvent is unlikely to change in the future anyway, heaven forbid that we're adding another Crowdsale contract with a differen event signature ;)